### PR TITLE
cleanup more unused variables

### PR DESCRIPTION
**Pull request by domsj**

### CODE REVIEW

File: src/paxos/master.ml [hunk 1]
```diff
@@ -257,6 +257,8 @@ let stable_master (type s) constants ((n,new_i, lease_expire_waiters) as current
 
 let master_dictate constants ms () =
   let {mo;v;n;i;lew} = ms in
+  ignore mo;
```
Comment (NicolasT) [inline src/paxos/master.ml]: Can't you not match on them instead?

Comment (domsj) [inline src/paxos/master.ml]: that gives another warning/error instead...

Comment (domsj) [inline src/paxos/master.ml]: I could write `let v,n,i = ms.v, ms.n, ms.i in` if you prefer that

Comment (NicolasT) [inline src/paxos/master.ml]: I thought `let { v; n; i; _ } = ms in` is supposed to work and not generate warnings/errors?

Comment (domsj) [inline src/paxos/master.ml]: You're right, updated.

**Comment by NicolasT** [comment]
LGTM

**Pull request merged**